### PR TITLE
test: add database testing.

### PR DESCRIPTION
**Pull request by seilagamo**
Add testing to the database layer

### CODE REVIEW

File: _script/setup [hunk 1]
```diff
@@ -0,0 +1,19 @@
+#!/bin/bash
+
+# Copyright 2021 Adevinta
```
Comment (manelmontilla) [inline _script/setup:3]: s/2021/2023

File: _script/setup [hunk 2]
```diff
@@ -0,0 +1,19 @@
...
+
+set -e
```
Comment (manelmontilla) [inline _script/setup:5]: Not a big deal but I would add the "-u" flag in the `set` command above, just to stop the execution of the script before using a variable that is not bound.

File: _script/start-pg [hunk 3]
```diff
@@ -0,0 +1,11 @@
+#!/bin/bash
+
+# Copyright 2021 Adevinta
+
+set -e
```
Comment (manelmontilla) [inline _script/start-pg:5]: Not a big deal but I would add the "-u" flag in the set command above, just to stop the execution of the script before using a variable that is not bound.

Comment (manelmontilla) [reply _script/start-pg:5]: Also I think there is an extra white space at the end of the line.

File: pkg/storage/postgresql/testdbutils.go [hunk 4]
```diff
@@ -0,0 +1,128 @@
...
+
+type mockLogger struct {
+	echo.Logger
+}
+
+func (l *mockLogger) Level() log.Lvl {
+	return log.OFF
+}
+
+// FromConnStrToDsn transform a struct with the connection data to a connection string
```
Comment (manelmontilla) [inline pkg/storage/postgresql/testdbutils.go:37]: Add a dot at the end of the line?

File: pkg/storage/postgresql/testdbutils.go [hunk 5]
```diff
@@ -0,0 +1,128 @@
+/*
+Copyright 2022 Adevinta
```
Comment (manelmontilla) [inline pkg/storage/postgresql/testdbutils.go:2]: s/2022/2023?

File: pkg/storage/postgresql/tickets_test.go [hunk 6]
```diff
@@ -4,17 +4,113 @@ Copyright 2023 Adevinta
...
+)
+
+var (
+	baseModelFieldNames = []string{"ID"}
+	ignoreFieldsTeam    = cmpopts.IgnoreFields(model.FindingTicket{}, baseModelFieldNames...)
+)
+
+func TestGetFindingTicket(t *testing.T) {
+
+	testStore, err := PrepareDatabaseLocal("../../../testdata/fixtures", NewDB)
```
Comment (manelmontilla) [inline pkg/storage/postgresql/tickets_test.go:24]: Also not a big deal but I would remove the first empty line above this one.

**Review by manelmontilla — CHANGES_REQUESTED** [review]

File: _script/setup (hunk 1 above)
Comment (seilagamo) [reply _script/setup]: Fixed in commit b0feeb3b4855a3d1fd63168e568883d82acee034

File: _script/setup (hunk 2 above)
Comment (seilagamo) [reply _script/setup]: Fixed in commit b0feeb3b4855a3d1fd63168e568883d82acee034

File: _script/start-pg (hunk 3 above)
Comment (seilagamo) [reply _script/start-pg]: Fixed in commit b0feeb3b4855a3d1fd63168e568883d82acee034

File: pkg/storage/postgresql/testdbutils.go (hunk 4 above)
Comment (seilagamo) [reply pkg/storage/postgresql/testdbutils.go]: Fixed in commit b0feeb3b4855a3d1fd63168e568883d82acee034

File: pkg/storage/postgresql/testdbutils.go (hunk 5 above)
Comment (seilagamo) [reply pkg/storage/postgresql/testdbutils.go]: Fixed in commit b0feeb3b4855a3d1fd63168e568883d82acee034

File: pkg/storage/postgresql/tickets_test.go (hunk 6 above)
Comment (seilagamo) [reply pkg/storage/postgresql/tickets_test.go:23]: Fixed in commit b0feeb3b4855a3d1fd63168e568883d82acee034

**Review by manelmontilla — APPROVED** [review]
LGTM

**Pull request merged**